lambdas/evaluate-gpg45-scores/src/test/java/uk/gov/di/ipv/core/evaluategpg45scores/EvaluateGpg45ScoreHandlerTest.java

```diff
@@ -284,7 +284,7 @@ void shouldReturn500IfFailedToParseCredentials() throws Exception {
 
     @Test
     void shouldReturn500IfCredentialOfUnknownType() throws Exception {
-        when(configurationService.getCredentialIssuer(any())).thenReturn(addressConfig);
+        //        when(configurationService.getCredentialIssuer(any())).thenReturn(addressConfig);
```

Comment:
Can this go?

Reply:
Oh yep 👍 it was just an unnecessary stubbing